script.js

```diff
@@ -0,0 +1,108 @@
+const inputText = document.querySelector('#todo-input');
```

Comment:
변수의 이름과 값이 정확하게 일치한다면 더 좋을것 같아요!
```suggestion
const inputText = document.querySelector('#todo-input').value;
```

Reply:
헉 이 생각을 못 했네요... 👍